src/lib/pipeline.ml

```diff
@@ -74,6 +74,7 @@ module Parameters = struct
     with_mutect2: bool [@default false];
     with_varscan: bool [@default false];
     with_somaticsniper: bool [@default false];
+    with_optitype: bool [@default false];
```

Comment:
To throw another monkey wrench into this but OptiType can run on both RNA and DNA